syntax = "proto3";
package payload.manager.admin.v1;

/*
  Instantaneous authoritative state snapshot.

  NOT time-series metrics (use OpenTelemetry for that).
*/
message StatsRequest {}

message StatsResponse {
  uint64 payloads_gpu = 1;
  uint64 payloads_ram = 2;
  uint64 payloads_disk = 3;

  uint64 bytes_gpu = 4;
  uint64 bytes_ram = 5;
  uint64 bytes_disk = 6;
}
